logic/scorer/scorer.go

```diff
@@ -15,6 +15,23 @@ func CribPoints(lead model.Card, crib []model.Card) int {
 	return points(lead, crib, true)
 }
 
+const cardsPerHand = 5
+
+// map card value (1 to 13) to number of occurrences in the hand
+// we have to include index 0 and index 14 here because some calculations pass those values in
+type valueToCount [15]uint8
+
+func (vtc valueToCount) get(value int) (uint8, bool) {
+	return vtc[value], vtc.containsKey(value)
+}
+
+func (vtc valueToCount) containsKey(key int) bool {
+	return vtc[key] > 0
+}
+
+// values is used to store a card's point value or rank value without allocating
+type values [cardsPerHand]int
```

Comment:
:cactus: initial impression: I don't like this type. once we start throwing the word "values" around everywhere, it loses its meaning. I'd rather see `[cardsPerScoringHand]uint8` everywhere (or a better type name) than `values`